service/raft/group0_state_machine.cc

```diff
@@ -184,6 +184,12 @@ future<> group0_state_machine::merge_and_apply(group0_state_machine_merger& merg
 future<> group0_state_machine::apply(std::vector<raft::command_cref> command) {
     slogger.trace("apply() is called with {} commands", command.size());
 
+    co_await utils::get_local_injector().inject("group0_state_machine::apply", [](auto& handler) -> future<> {
```

Comment:
Please use better name for injection point that also describes what it does (sleeps). Also a nit: there should be a space between [] and (.

Reply:
Will "group0_state_machine::delay_apply" be a good name? Also fixed the lambda, in fact I found another example of the timeout and it looks like I don't need to use lambda at all, just to provide the delay as duration.

(the space between [] and ( has been removed by the clang-format that I tried to setup ... looked into the options, but apparently there is no option in the clang-format that would keep the space in between these :disappointed: )

Reply:
> Will "group0_state_machine::delay_apply" be a good name?

Yes.

> Also fixed the lambda, in fact I found another example of the timeout and it looks like I don't need to use lambda at all, just to provide the delay as duration.

Right.